sklearn/plot/tests/test_heatmap.py

```diff
@@ -0,0 +1,24 @@
+from sklearn.plot import plot_heatmap
+from sklearn.utils.testing import SkipTest
+import numpy as np
+
+
+def test_heatmap():
+    try:
+        import matplotlib
+    except ImportError:
+        raise SkipTest("Not testing plot_heatmap, matplotlib not installed.")
+
+    import matplotlib.pyplot as plt
```

Comment:
nope these are tricky to resolve. it requires me to re-install python using a different method and then i'll have to setup my scikit-learn development environment again. since sklearn doesn't test a lot on matplotlib, i'm just using a jupyter-notebook for this PR.